Dockerfile

```diff
@@ -118,7 +118,7 @@ COPY root /
 
 RUN chmod +x /usr/local/bin/run.sh /usr/local/bin/occ /etc/periodic/15min/nextcloud
 
-VOLUME ["/data","/opt/nextcloud"]
+VOLUME ["/data"]
```

Comment:
I had made this change locally at one point for testing but rolled it back to what you have for consistency. In other words, this line matches master.